knowledge_base/save_job_result_to_volume/resources/my_volume.volume.yml

```diff
@@ -0,0 +1,9 @@
+resources: 
+  volumes: 
+    my_volume:
+      catalog_name: main
+      name: my_volume
+      # We use the ${resources.schemas...} interpolation syntax to force the creation
+      # of the schema before the volume. Usage of the ${resources.schemas...} syntax 
+      # allows Databricks Asset Bundles to form a dependency graph between resources.
```

Comment:
Do we need to go into this here?

Reply:
We have had multiple SAs reach out to us and ask how to sequence the resource creation. Given that folks will often use schemas with their volumes it feels relevant to keep this bit here.